client/FxAccountClient.js

```diff
@@ -1071,6 +1077,45 @@ define([
       });
   };
 
+  /**
+   * Send an unblock code
+   *
+   * @method sendUnblockCode
+   * @param {String} email email where to send the login authorization code
+   * @return {Promise} A promise that will be fulfilled with JSON `xhr.responseText` of the request
+   */
+  FxAccountClient.prototype.sendUnblockCode = function (email) {
+    required(email, 'email');
+
+    var data = {
+      email: email
+    };
+
+    var request = this.request;
```

Comment:
I noticed this also, but then expanding the file, I notice this pattern in many of the methods, so felt it was just for consistency. Should they all be changed (different PR)?